packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -3,7 +3,7 @@ import styled from '@emotion/styled';
 import { isNonEmptyString } from '@sniptt/guards';
 import { useRecoilState, useRecoilValue } from 'recoil';
 import { Key } from 'ts-key-enum';
-import { Avatar, IconNotes } from 'twenty-ui';
+import { Avatar, IconNotes, IconSparkles } from 'twenty-ui';
```

Comment:
Of course the sparkles, no AI without sparkles ✨!

Reply:
:D